smoke-test/src/test/java/org/opennms/smoketest/minion/SyslogKafkaElasticsearchBufferingIT.java

```diff
@@ -63,6 +64,7 @@
  * 
  * @author Seth
  */
+@Ignore
```

Comment:
I think you may need to add the new `opennms-osgi-core-rest` feature to `https://github.com/OpenNMS/opennms/blob/1c8c7b8da26320fa2cc4ad0a764c72597bfc4fea/smoke-test/src/test/resources/org.apache.karaf.features.cfg#L87` to get this one to work.